server/main.py

```diff
@@ -94,3 +122,28 @@ async def calculate_spectrum(payload: Payload):
                 "units": spectrum.units[payload.mode],
             },
         }
+
+#[Date] _[database] _[molecule]_[temperature]K_[pressure]atm
+@app.get("/download_spectrum")
+async def download_spec(payload: Payload):
+    print(payload)
+    date = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S");
+    file_name = f"create_spectrum/{date}_{payload.database}_{payload.tgas}k_{payload.pressure}atm.spec"
```

Comment:
I think this name is going to be better but if you want i can change.

Reply:
You're missing the molecule.

Reply:
molecule is kind of list attribute  so it is going to create some problems